src/components/frame/Extensions/layouts/SharedLayout.tsx

```diff
@@ -43,6 +46,16 @@ const SharedLayout: FC<PropsWithChildren<SharedLayoutProps>> = ({
 
   const chainIcon = getChainIconName(chainId);
 
+  useLayoutEffect(() => {
+    if (
+      (!wallet || isBasicWallet(wallet)) &&
```

Comment:
This will try to perform background login even if there's no saved info in local storage. Should be:

``` js
useLayoutEffect(() => {
    if ((!wallet && getLastWallet()) || isBasicWallet(wallet)) {
      connectWallet?.();
    }
  }, [connectWallet, wallet]);
  ```